src/opf/dcp.jl

```diff
@@ -159,14 +157,18 @@ function extract_dual(opf::OPFModel{DCOPF})
         dual_solution["va_diff"] = dual.(model[:va_diff])
 
         # Duals of variable lower/upper bounds
-        # bus
-        # N/A
+        # We store λ = λₗ + λᵤ, where λₗ, λᵤ are the dual variables associated to
+        #   lower and upper bounds, respectively.
+        # Recall that, in JuMP's convention, we have λₗ ≥ 0, λᵤ ≤ 0, hence
+        #   λₗ = max(λ, 0) and λᵤ = min(λ, 0).
+
+        # (no bounded bus-level variables)
         # generator
-        dual_solution["pg_lb"] = dual.(LowerBoundRef.(model[:pg]))
-        dual_solution["pg_ub"] = dual.(UpperBoundRef.(model[:pg]))
+        dual_solution["pg"] .+= dual.(LowerBoundRef.(model[:pg]))
+        dual_solution["pg"] .+= dual.(UpperBoundRef.(model[:pg]))
```

Comment:
This syntax is correct because we initialized the dual value to be a correctly-shaped array of zeros.
The downside is that it makes it less obvious what the final product is.

Reply:
Yep, not convinced, I'll change it. I think the improved legibility is worth the extra memory allocations